sample/src/main/java/com/polidea/cockpit/sample/apipresentation/ApiPresentationBasePresenter.kt

```diff
@@ -0,0 +1,22 @@
+package com.polidea.cockpit.sample.apipresentation
+
+import android.graphics.Color
+import com.polidea.cockpit.cockpit.Cockpit
+
+abstract class ApiPresentationBasePresenter(open val apiPresentationView: ApiPresentationBaseContract.View<*>) : ApiPresentationBaseContract.Presenter {
+
+    override fun start() {
+        initViews()
```

Comment:
Here is the only place `initViews()` method is called. So there is no reason to expose it to `ApiPresentationBaseContract.Presenter`. It could be simply `protected open fun initViews()` method